src/Localization/en/translations.json

```diff
@@ -24,8 +26,14 @@
             "Collection Throughput": "Collection Throughput",
             "Enable DB level throughput": "Enable DB level throughput",
             "Database Throughput": "Database Throughput",
-            "RefreshMessage": "Self Serve Example successfully refreshing",
-            "SubmissionMessage": "Submitted successfully"
+            "UpdateInProgressMessage": "Self Serve Example is being updated",
+            "UpdateCompletedMessageTitle":"Update Succeeded",
+            "UpdateCompletedMessageText":  "Self Serve Example update completed",
+            "SubmissionMessageSuccessTitle": "Submission Succeeded",
+            "SubmissionMessageForNewRegionText": "Submitted successfully. Region Changed!",
+            "SubmissionMessageForSameRegionText": "Submitted successfully. Region Unchanged!",
+            "SubmissionMessageErrorTitle": "Submission Failed",
+            "SubmissionMessageErrorText": "Submitted Failed because of errors"
```

Comment:
>Submitted [](start = 43, length = 9)

Please use better strings. Submit failed. No upper case in Failed, Region Unchanged, etc